### PR TITLE
MEAN DAO TVL

### PR DESCRIPTION
**Pull request by supermean-ayaz**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Mean DAO

##### Twitter Link:
https://twitter.com/meanfinance

##### List of audit links if any:
https://docs.meanfi.com/products/safety-and-security/#_audits

##### Website Link:
www.meanfi.com

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
All logo variants are [here](https://supermean.notion.site/Logos-53012f81cf6c4e3da8818134ab79e16c)

##### Current TVL:
31,530,660.84 USD

##### Chain:
Solana

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
[meanfi](https://www.coingecko.com/en/coins/meanfi)

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
[meanfi](https://coinmarketcap.com/currencies/meanfi/)

##### Short Description (to be shown on DefiLlama):
Mean DAO is the protocol DAO powering MeanFi, a censorship-resistant, user-friendly, self-custody, permissionless & trustless bank bringing everyday banking workflows and real-time finance to crypto and DeFi.

##### Token address and ticker if any:
`MEANeD3XDdUmNMsRGjASkSWdC8prLYsoRJ61pPeHctD`

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Payments

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
None

##### forkedFrom (Does your project originate from another project):
We forked from this repo

##### methodology (what is being counted as tvl, how is tvl being calculated):
Current value of tokens locked in the protocols in payment streams, vesting, treasuries and super safes.

### CODE REVIEW

File: projects/meanfi.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    const tvlSnapshotResponse = await axios.get(
+        "https://raw.githubusercontent.com/mean-dao/MEAN-stats/main/mean-stats.json"
+    );
+    const { data } = tvlSnapshotResponse;
+    return data.tvl.total;
+}
+
+module.exports = {
+    timetravel: false,
+    methodology: "Snapshots of the TVL from app.katana.so are saved periodically into the statistics repo",
```
Comment (micheltriana) [inline projects/meanfi.js:13]: What's this note here about Katana?

**Review by micheltriana — APPROVED** [review]
Looks good.

**Comment by llamatester** [comment]
The adapter at projects/mean-dao.js exports TVL: 
        
 
 
    
    total                    29.97 M

**Comment by g1nt0ki** [comment]
@supermean-ayaz thanks for the PR, sorry but we no longer accept fetch adapters for new listings. Can you please compute tvl using on chain calls? 

Is super safe/treasury like gnosis safe? if so, not sure if it should be included in tvl.

We dont count vested tokens as tvl of any protocol (eg: llama, team finance, pinksale, parallel DaoFi), reason for this is, those tokens are not yet in circulation and can be used for inflating tvl

payment streams are good, are the counted tokens locked in steaming contract?

**Comment by g1nt0ki** [comment]
@supermean-ayaz @micheltriana we have added a new export key called 'vesting' for non-stablecoin tokens specifically for payments/streaming category

**Comment by g1nt0ki** [comment]
closing this due to lack of activity

**Pull request closed without merging**